smoke-tests/test/npm-replace-global.js

```diff
@@ -136,6 +136,8 @@ t.test('publish and replace global self', async t => {
   if (setup.SMOKE_PUBLISH) {
     await npmPackage()
   }
+  registry.nock.get('/npm').reply(404, 'not found')
+  registry.nock.get('/npm').reply(404, 'not found')
```

Comment:
This is the npm outdated check that npm does, right?  This technically may or may not happen during an npm request.  I don't think it's likely during the current smoke tests but we'd probably be doing ourselves a favor if we mocked this in a way that showed that it is optional.  nock has `optionally` and `persist` and I think they work together.